### PR TITLE
fix #131, fix #132. rewrite ending of abstract. fix intro figure

### LINKED ISSUE #131: Remove/rewrite abstract ending

### LINKED ISSUE #132: Improve introduction figure

### PR DESCRIPTION
**Pull request by SebastianMygind**

### CODE REVIEW

File: src/introduction/introduction.tex
Comment (audio-engineer) [inline src/introduction/introduction.tex]: This cannot be correct, right?

File: src/introduction/introduction.tex
Comment (audio-engineer) [inline src/introduction/introduction.tex]:
```suggestion
These tools are essential to understanding the complexity of mid- and endgame strategies of chess.
```

**Review by audio-engineer — DISMISSED** [review]

File: src/main.tex
Comment (Danar435) [inline src/main.tex]: Spelling mistake, remove 'is'

**Review by Danar435 — CHANGES_REQUESTED** [review]

File: src/main.tex
Comment (TexWO) [reply src/main.tex]: line 70 btw

File: src/introduction/introduction.tex
Comment (Danar435) [reply src/introduction/introduction.tex]: I think that "mid- to endgame" sounds better

**Review by TexWO — APPROVED** [review]

**Review by Danar435 — APPROVED** [review]

**Comment by SebastianMygind** [comment]
@audio-engineer I have implemented your changes will dismiss review to not block merge.

**Pull request merged**
